gluu-server/install/community-edition-setup/templates/jetty/fido2

```diff
@@ -10,7 +10,7 @@ TMPDIR=%(jetty_dist)s/temp
 
 export PYTHON_HOME=%(jython_home)s
 
-JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/opt/azure/applicationinsights-agent-3.4.14.jar"
+JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/opt/azure/applicationinsights-agent-3.4.16.jar"
```

Comment:
Considering the same package is used in multiple template files, its an opportunity to create a common file or read in the JAVA_OPTIONS from a common config file